data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -16,30 +16,37 @@
-            music=the_city_falls.ogg
+            music=elvish-theme.ogg
             # po: If the target language has a specialized term for a fortified town, like Latin "oppidum", use it for Parthyn.
-            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the baron of the city. Every summer when the mountain passes became clear, orcs came down from the north to prove themselves in battle. Every year Parthyn’s guards repelled the raids, though some of the townsfolk were always lost. Malin’s father led raiding parties to disrupt the orc encampments and send the orcs back to the north."
+            story= _ "Malin Keshar was born ten years after the death of Haldric IV. He grew up in the northern border town of Parthyn, the second child and eldest son of the baron of the city. Every summer, by the time the mountainous paths and high passes shed their wintry gowns of snow, orcs descended from their northern hills to ravage the settlements at the frontier. Every year, Malin’s father led the townsfolk to repel the raids and force the orcs to retreat back to the north."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            story= _ "When Malin was 10, a mage who passed through the town sensed strong magical abilities in the boy. His parents, seeing a chance for him to live away from the constant battles with the orcs, arranged for him to be sent to the mage community on the Isle of Alduin. There he studied for eight years, while back at home the orc raids increased in their frequency and ferocity. Before Malin could finish his training, however, he was expelled."
+            story= _ "When Malin was 10, a mage who passed through the town sensed strong magical abilities in the boy. His parents, seeing a chance for him to escape the constant war with the orcs, arranged for him to be sent to the mages at the Isle of Alduin. He studied there for eight years, learning all manner of magic, from basic control over the elements, to the mystic arts and scrying, to alchemy and simple medicine."
             {STORYTXT_BACKGROUND_BIGMAP_DID}
         [/part]
         [part]
-            story= _ "Malin returned to Parthyn just as summer began only to encounter more misfortune. Even as he arrived, an early, unexpected orc raid was being conducted against Parthyn. Malin watched as his father took an arrow in the chest and fell lifeless to the ground."
+            story= _ "Malin quickly found that he had considerable talent, especially for arcane sorcery and manipulating the essence of life. In time, however, he grew bored of his studies and began to research magics that were far beyond the teachings of his masters. Despite Malin’s substantial skill, his lack of experience led to a series of failed experiments, one of which took the life of a fellow student. For his transgressions, he was expelled."
```

Comment:
I don't think it improves the story to spill the beans on the reasons of his expulsion right there in the intro text. Previously, it was simply mentioned in passing that he was expelled, with the actual and perfectly understandable reason finally revealed when he gets driven out of Parthyn for good. Also, part of the effectiveness of the original reveal is that everything that has gone wrong since has gone wrong because of a relatively small incident. With these changes, there is no reveal because the intro already tells what Malin did (he got someone killed), and it's much less of a "small mistake with big consequences".

Also, being good at manipulation of life, growing bored of mundane mage studies and conducting dangerous experiments are the hallmark cues for "this guy wants to be a necromancer", which comes off as hitting the player over the head with the fact that the story is obviously going to be about necromancy (they already know that, there's no need to repeat the standard tropes associated with that).

I'd suggest even avoiding explicitly stating that Malin has considerable **talent** in magic (which is slightly different than "strong magical abilities" above), since to me being at best average is part of his characterization. He shouldn't have gotten expelled because he was basically outgrowing the academy's teachings, but simply because he was impulsive and ill-tempered. Not every powerful mage/necromancer should start out with "considerable talent", and I think Malin is a good example of someone whose story benefits from him having been distinctly average rather than exceptional.

Reply:
When I read it over the first time, I found myself wondering why he got expelled, but the story gave no indication why until S6, by which time I didn't really care anymore. I think it's actually better to say it early because it's not even much of a reveal at all (it's just a little, tiny thing) and it will make the reader wonder what in the world happened (you can't just say he got expelled without reason). 

In the original, I didn't see him being average as part of his characterization. He learned much faster than your average mage under Darken Volk, who also praises his talent and ability (and potential). He was always a very strong mage, so this isn't something I added (merely stated in the intro text).

Reply:
I know I'm late to the party here, but I agree with zookeeper that this is an important reveal you're now spoiling before the first scenario even starts.